### PR TITLE
PLU-275: [BTN-MIGRATION-3] Introduce info message for Postman SMS

### PR DESCRIPTION
**Pull request by ogp-weeloong**
# Problem
We want to make it clear to users that our SMS action is only free until a certain date.

# Solution
Users may not read our guidebook, so we will put this message inside our UI (where there is a higher chance of being read)

This PR introduces a new "setup message" attribute to display such messages.

# Tests
- Check that info message shows up when Postman SMS is selected
- Regression test: check that info message does not show up for other apps.

### CODE REVIEW

**Comment by ogp-weeloong** [comment]
* **#607** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/607?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#616](https://github.com/opengovsg/plumber/pull/616) <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/616?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#625](https://github.com/opengovsg/plumber/pull/625) <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/625?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* **#606** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/606?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#605** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/605?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ogp-weeloong and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pregnantboy — APPROVED** [review]
lgtm

File: packages/frontend/src/components/MarkdownRenderer/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ interface MarkdownRendererProps {
 }
 
 export default function MarkdownRenderer(props: MarkdownRendererProps) {
-  const { allowMultilineBreaks = true, source } = props
+  const { allowMultilineBreaks = true, source, components = {} } = props
```
Comment (m0nggh) [inline packages/frontend/src/components/MarkdownRenderer/index.tsx:30]: wah nice u helped me to add back the customizability part thks

Comment (ogp-weeloong) [reply packages/frontend/src/components/MarkdownRenderer/index.tsx:30]: anything for you

**Comment by ogp-weeloong** [comment]
### Merge activity

* **Jun 14, 5:49 AM EDT**: @ogp-weeloong started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/607).
* **Jun 14, 5:51 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/607) rebased this pull request as part of a merge.
* **Jun 14, 5:51 AM EDT**: @ogp-weeloong merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/607).

**Pull request merged**